Makefile

```diff
@@ -34,7 +34,7 @@ migrate-rollback:
 	npx knex migrate:rollback --knexfile './dist/knexfile.js'
 
 start-dev:
-	npx tsc-watch --onSuccess 'node dist/app.js'
+	npx tsc-watch --onSuccess 'node --inspect=9339 dist/app.js'
```

Comment:
Are there any drawbacks to this?

Reply:
nope. only benefits

Reply:
the only drawback I can think of is if someone is running multiple node apps and has already assigned a debugger to port `9339` it will complain (default is `9229`).